dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/resource/entity/FileResource.java

```diff
@@ -217,4 +220,13 @@ public Response getExport(@ApiParam(value = "Set Id", required = true) @PathPara
     return fileService.getIndexMetadata();
   }
 
+  @POST
+  @Path("/summary/manifest")
+  @Timed
+  @Consumes(APPLICATION_JSON)
+  @Produces(APPLICATION_JSON)
+  public Map<String, Map<String, Long>> getManifestSummary(ManifestSummaryQuery summary) {
+    return fileService.getManifestSummary(summary);
```

Comment:
I wonder if this really has anything to do with manifests. I realize the use case is manifests, but is it from a resource perspective? Seems odd that it is in the `FileService`. Seems like it should be something like `getPrioritizedUniquenessSummary` or something...